csharp/OpenStudio.nuspec.in

```diff
@@ -5,19 +5,27 @@
     <version>@OpenStudio_VERSION@</version>
     <authors>NREL</authors>
     <owners>NREL</owners>
-    <licenseUrl>http://github.com/NREL/OpenStudio/blob/develop/LICENSE.md</licenseUrl>
+    <license type="file">LICENSE.md</license>
     <projectUrl>http://github.com/NREL/OpenStudio</projectUrl>
     <description>Whole building energy simulation</description>
     <tags>OpenStudio EnergyPlus Simulation Modeling</tags>
-    <references>
-        <reference file="OpenStudio.dll" />
-    </references>
+    <dependencies>
+      <group targetFramework=".NETFramework3.5" />
+    </dependencies>
   </metadata>
   <files>
-    <file src="@OPENSTUDIO_CSHARP_DLLNAME@" target="lib" /> <!-- TODO: target should be lib\{framework}, eg `lib\netcoreapp3.0` -->
-    <file src="@OPENSTUDIO_CSHARP_LIBNAME@" target="build" /> <!-- target to 'build' is probably not right either -->
-    <file src="@OPENSTUDIO_MODEL_CSHARP_LIBNAME@" target="build" />
-    <file src="@OPENSTUDIO_TRANSLATORS_CSHARP_LIBNAME@" target="build" />
-    <file src="@PROJECT_SOURCE_DIR@/csharp/OpenStudio.targets" target="build" /> <!-- To copy native binaries -->
+    <file src="Products/csharp/$CONFIGURATION$/OpenStudio.dll" target="lib/net35" />
```

Comment:
@tijcolem  $CONFIGURATION might need to be removed via configuration to make ninja work.

Reply:
yeah exactly. We might not to configure the presence of $CONFIGURATION$ LOL